### ISSUE TITLE
support ISO space-seperator

### ISSUE THREAD

**Issue by iamziyue** [spencermountain/spacetime#42]
Hi, I tested the following code：
~~~
console.log(spacetime('2018-02-02T22:00:00'))
console.log(spacetime('2018-02-02 22:00:00'))
~~~
The first line, I get:
~~~
{tz: "Asia/Shanghai", valid: true, epoch: 1517580000262}
~~~
The second line, I get:
~~~
Warning: couldn't parse date-string: '2018-02-02 22:00:00'
u @ spacetime.js?be0a:4
u @ spacetime.js?be0a:5
i @ spacetime.js?be0a:4
submitDiary @ Index.vue?2753:49
boundFn @ vue.esm.js?efeb:190
click @ Index.vue?1a5d:176
invoker @ vue.esm.js?efeb:2004
fn._withTask.fn._withTask @ vue.esm.js?efeb:1802
Index.vue?2753:49 u {tz: "Asia/Shanghai", valid: false, epoch: null}
~~~
Actually, for date strings, I use spaces as separators more often than ever. Do you think you can add this feature? Thank you.

**Comment by spencermountain** [comment]
hi @iamziyue. yeah, neat idea. I like that idea.
a regex-switch [here](https://github.com/smallwins/spacetime/blob/master/src/input/strParse.js#L72) should do it maybe.
I can look at it for the next release.
cheers

**Comment by spencermountain** [comment]
got this fixed in `5.1.1`
thanks.
